website/util/sanitize.py

```diff
@@ -11,6 +11,10 @@ def strip_html(unclean):
     :return: stripped string
     :rtype: str
     """
+    # We make this noop for non-string, non-collection inputs so this function can be used with higher-order
+    # functions, such as rapply (recursively applies a function to collections)
+    if not isinstance(unclean, basestring) and not is_iterable(unclean) and unclean is not None:
```

Comment:
As discussed with @sloria: this is ok for now due to apiv2 issues, but realistically we shouldn't be blindly using `strip_html` with `rapply` in the first place. 

This is definitely a hack to support a problematic use case. Current handling of sanitization in apiv2 is something that could stand to be improved, but that would require someone to file a separate ticket, and may not make it into the initial release regardless.